public/components/SelectDataTable.vue

```diff
@@ -128,7 +128,7 @@ export default Vue.extend({
   filters: {
     /* Display number with only two decimal. */
     cleanNumber(value) {
-      return parseFloat(value).toFixed(2);
+      return _.isNumber(value) ? value.toFixed(2) : "—";
```

Comment:
Changes non available data to display a dash instead of **NaN**.
I choose a dash to make clear that we do not have any data, instead of missing data, as those are calculated information.

Reply:
![Screen Shot 2020-09-01 at 15 38 46](https://user-images.githubusercontent.com/636801/91898199-60531400-ec69-11ea-8e93-19bd88f3e6a8.png)